pkg/config/autodiscover/autodiscover_targets.go

```diff
@@ -260,6 +263,31 @@ func getConfiguredOperatorTests() []string {
 	return opTests
 }
 
+// setBundleAndIndexImage provides the bundle image and index image for a given CSV.
+// These variables are saved in the `configsections.Operator` in order to be used by DCI,
+// which obtains them from the claim.json and provides them to preflight suite.
+func setBundleAndIndexImage(csvName, csvNamespace string) (bundleImage, indexImage string) {
```

Comment:
Could we have unit tests to cover this function?

Reply:
Sure, I'll take a look, I'm waiting for having some resources ready in our labs to do some testing

Reply:
@sebrandon1 , now I remember that I am calling this method when isTest variable is set to false (https://github.com/test-network-function/test-network-function/pull/612/files#diff-25fc9547da0d403308471a07947597646a4691b31fbecda834a8d157183da470L240), just for simplicity. So, what do you recommend for doing unit tests for this function?

Reply:
Sure thing, so in `pkg/config/autodiscover/autodiscover_targets_test.go` we should have a function called `TestSetBundleAndIndexImage()` that has some expected inputs and outputs and tests normal operation plus edge cases.

Something like [this](https://github.com/test-network-function/test-network-function/blob/main/test-network-function/accesscontrol/suite_test.go#L31-L70) unit test from the accesscontrol package for example.